Banking_System/src/CheckingAccount.java

```diff
@@ -1,28 +0,0 @@
-public class CheckingAccount extends Account{
-
-    static private int totalChecksIssued; //total number of checks issued across all customers
```

Comment:
It's understandable and clear why you defined it static, however, I think it makes better sense here to define it instance and return the number issued per object.